blockchains/tezos/validators/list.json

```diff
@@ -10,5 +10,11 @@
         "name": "Crypto Delegate",
         "description": "CryptoDelegate delegation service is the best alternative to running your own node. The easy way to grow your Tezos stake is to Delegate your XTZ Tezzies.",
         "website": "https://www.cryptodelegate.com"
+    },
+    {
+        "id": "tz2FCNBrERXtaTtNX6iimR1UJ5JSDxvdHM93",
+        "name": "🐠stake.fish",
```

Comment:
can you also add an image for tezos?

Reply:
Added here https://github.com/trustwallet/assets/commit/903268fb4174b2d4aa3108978cc1826657c82cb9